src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -783,7 +785,7 @@ $DOCKERFILE_SIGIL
 
     // Verify the dockerfile.
     if (verifyDockerfileOnBuild.value) {
-      val isUnchanged = checkDockerfile.value
+      val isUnchanged = checkDockerfile.value: @sbtUnchecked
```

Comment:
so did this used to not need `@sbtUnchecked` but now needs it?

Reply:
it was warning before, but this explicitly silences the warning because the publish step was exiting on warnings. That's also why I moved some of the loggers in here around.